src/DatePicker/index.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import {
+  default as AntDatePicker,
+  DatePickerProps as AntDatePickerProps,
+  RangePickerProps as AntRangePickerProps
+} from 'antd/lib/date-picker'
+
+import '../assets/style/form-item.css'
+import 'antd/lib/style/default.css'
+import 'antd/lib/date-picker/style/index.css'
+import 'antd/lib/empty/style/index.css'
```

Comment:
There is a dependency on the empty component. Will need to include the new Empty component styles here